app/views/users/new.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1><%= t "user" %></h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<% provide :title, I18n.t("signup") %>
```

Comment:
In this section, you can study about more how you can use I18n in views
[http://guides.rubyonrails.org/i18n.html#abstracting-localized-code](http://guides.rubyonrails.org/i18n.html#abstracting-localized-code)